numpy/lib/twodim_base.py

```diff
@@ -439,18 +439,20 @@ def tril(m, k=0):
     Lower triangle of an array.
 
     Return a copy of an array with elements above the `k`-th diagonal zeroed.
+    For arrays with `ndim` exceeding 2, `tril` will apply to the final two
```

Comment:
```suggestion
    For arrays with ``ndim`` exceeding 2, `tril` will apply to the final two
```

Just an rst nit. You could also reword if you prefer, e.g. "For arrays with greater than 2 dimensions..."

Reply:
Done, thanks!